### DISCUSSION TITLE
Is it possible to draw an annotation only if it is within a date range?

### DISCUSSION THREAD

**Discussion by geerdes** [chartjs/chartjs-plugin-annotation#692] [Q&A] [answered]
I am trying to mark a certain period with a box. I have a dynamic chart where I can specify a date range. The x-axis is of type 'time'. But if my date range is before or after the annotation, the chart is always extended up to the specified xMin/xMax values of the annotation. However, I do not want the annotation to extend the chart.

**Comment by kurkle** [comment, marked as answer]
`adjustScaleRange` option is what you are looking for.

**Reply by geerdes** [reply]
Thank you so much
